### PR TITLE
DOCSP-5399 Apply template to node driver landing page [No merge]

### PR DESCRIPTION
**Pull request by cbush**
Please ignore the Java parts. This PR builds on https://github.com/mongodb/docs-ecosystem/pull/493 and will be rebased when that PR is merged.

### JIRA
https://jira.mongodb.org/browse/DOCSP-5399

### Staged
https://docs-mongodbcom-staging.corp.mongodb.com/ecosystem/bush/docsp-5399-node-landing/drivers/node.html

### CODE REVIEW

File: source/drivers/java.txt [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+- `Usage Guide <http://mongodb.github.io/mongo-java-driver/3.10/driver/>`__
+
+- `API Documentation <http://mongodb.github.io/mongo-java-driver/3.10/javadoc/overview-summary.html>`_
+
+- `Changelog <http://mongodb.github.io/mongo-java-driver/3.10/whats-new/>`__
+
+- `Source Code <https://github.com/mongodb/mongo-java-driver/>`__
+
+
```
Comment (skerschb) [inline source/drivers/java.txt]: If we link to the Connect guide (docs.mongodb.com/guides/drivers) here we get the dual benefit of taking the burden of connection tutorials off the driver page and also providing an incentive to make sure the guide stays current (the more we duplicate content, the more likely it is to get out of date somewhere on the site).

File: source/drivers/java.txt [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+Connect to a Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a :manual:`replica set </replication>`, you must specify one or more members
+to the MongoClient constructor:
+
+.. code-block:: java
+
+   MongoClient mongoClient = new MongoClient(
+       new MongoClientURI("mongodb://host1:27017,host2:27017,host3:27017"));
```
Comment (skerschb) [inline source/drivers/java.txt]: There are a few caveats to putting code examples on a drivers landing page.  The content is uncategorized and the landing page is likely to change, and connection examples are frequently bookmarked. What we might do instead is link out to the existing old page in a clear way. Or provide a link to a separate page with the connection examples on it (hopefully single-sourced somewhere and integrated with the driver's tests).

Comment (cbush) [reply source/drivers/java.txt]: @terakilobyte thoughts on this?

**Comment by cbush** [comment]
Moved to https://github.com/mongodb/docs-ecosystem/pull/497

@skerschb I have removed the other connection examples and left only the Atlas connection snippet, which can be found in Atlas anyway.

**Pull request closed without merging**